xml/ha_migration.xml

```diff
@@ -562,24 +601,84 @@
 &prompt.root;systemctl enable sbd</screen>
     </step>
    </procedure>
-   <para/>
-   <note xml:id="note.ha.cib.upgrade">
+
+ <note xml:id="note.ha.cib.upgrade">
     <title>Upgrading the CIB Syntax Version</title>
     <para>
-     Tags (for grouping resources) and some ACL features only work with the
-     CIB syntax version <literal>pacemaker-2.0</literal> or higher. (To
-     check your version, use the <command>cibadmin -Q |grep
-     validate-with</command> command.) If you have upgraded from &productname; 11
-     SPx, your CIB version will <emphasis>not</emphasis> be upgraded by
-     default. To manually upgrade to the latest CIB version use one of the
-     following commands:
-    </para>
+     Sometimes new features are only available with the latest CIB syntax version.
+     Depending from which product version you upgraded, your CIB syntax version
+     will <emphasis>not</emphasis> be upgraded by default.
+     </para>
+    <procedure>
+     <step>
+      <para>Check your version with:</para>
+<screen>cibadmin -Q | grep validate-with</screen>
+     </step>
+     <step>
+      <para>Manually upgrade to the latest CIB syntax version with:
+      </para>
 <screen>&prompt.root;<command>cibadmin</command> --upgrade --force</screen>
-    <para>
-     or
-    </para>
-<screen>&prompt.root;<command>crm</command> configure upgrade force</screen>
+     </step>
+    </procedure>
    </note>
+
+<procedure xml:id="pro.ha.migration.offline.12-15">
+    <title>Upgrading from Product Version 12 to 15: Cluster-Wide Offline Migration</title>
+   <important>
+     <title>Installation from Scratch</title>
+     <para>If you decide to install the cluster nodes from scratch (instead
+      of upgrading them), see <xref linkend="sec.ha.requirements.sw"/> for the
+      list of modules required for &productname; &productnumber;. Find more
+      information about modules, extensions and related products in the release
+      notes for &sls; 15. They are available at <link
+       xlink:href="https://www.suse.com/releasenotes/"/>.
+     </para>
+    </important>
+    <step>
+     <para>
+      Before starting the offline migration to &productnumber; 15, manually
```

Comment:
Remove 15, it's already included in the `&productnumber;` entity.

Reply:
Argh, definitely yes.